docs/userGuide/syntax/lists.md

```diff
@@ -109,3 +109,98 @@ first number
 - [x] Item 4
 - ( ) Item 5
 </div>
```

Comment:
If committing image files, let's not use higher resolution than necessary, to reduce repo size.